lib/src/components/NxTransferListHalf/__tests__/NxTransferListHalf.test.tsx

```diff
@@ -168,6 +168,54 @@ describe('NxTransferListHalf', function() {
     expect(items[1]).toHaveTextContent('baz');
   });
 
+  it('render from 0 to 3 items', function() {
+    const view = quickRender({ items: [] });
+    const itemList = view.container.querySelector('.nx-transfer-list__item-list') as HTMLElement,
+        items = itemList.querySelectorAll('.nx-transfer-list__item');
+    expect(items.length).toBe(0);
```

Comment:
```suggestion
    const view = quickRender({ items: [] });
    const el = view.container;
    expect(within(el).queryByRole('group')).not.toBeInTheDocument();
```

Reply:
it does not work, an existing group seems to already exist

![Screenshot 2024-11-21 181849](https://github.com/user-attachments/assets/ef9a42d9-8298-4bc6-aadb-037fd6a3de70)

Reply:
> it does not work, an existing group seems to already exist

Ah I meant `within(view.container.firstElementChild)`.  I was thinking `container` was the `.nx-transfer-list` but it's not, it's a div that RTL creates and renders the `.nx-transfer-list` within.

Reply:
> > it does not work, an existing group seems to already exist
> 
> Ah I meant `within(view.container.firstElementChild)`. I was thinking `container` was the `.nx-transfer-list` but it's not, it's a div that RTL creates and renders the `.nx-transfer-list` within.

done at 2235ba0b934be804d56e26ccaf80c7b041f94abc